package.json

```diff
@@ -46,11 +46,16 @@
     "geist": "^1.3.1",
     "gpt-tokenizer": "^2.8.1",
     "gpt4-tokenizer": "^1.3.0",
+    "highlight.js": "^11.11.1",
+    "html-react-parser": "^5.2.3",
     "html2pdf.js": "^0.10.2",
     "joi": "^17.13.3",
     "langchain": "^0.3.19",
     "loops": "^3.4.0",
     "lucide-react": "^0.471.0",
+    "marked": "^15.0.11",
```

Comment:
why are you adding a new markdown package instead of using `react-markdown` which is used by
- Vercel AI SDK team (see chat sdk): https://github.com/vercel/ai-chatbot/blob/main/package.json 
- existing Recoup chat components
-